test/elm/comparison/data.coffee

```diff
@@ -6237,14 +6453,7 @@ module.exports['Equivalent'] = {
             } ],
             "expression" : {
                "localId" : "12",
-               "type" : "Equivalent",
-               "operand" : [ {
-                  "localId" : "10",
-                  "type" : "Null"
-               }, {
-                  "localId" : "11",
-                  "type" : "Null"
-               } ]
+               "type" : "Null"
```

Comment:
You can see here that since CQL-to-ELM now chokes on `null ~ null`, it's just exporting a literal `null` (not even calling `Equivalent`).  So... this test is pretty much useless as-is.

If you change the test to something like `(null as String) ~ (null as String)` then hopefully the ELM would actually invoke the `Equivalent` operator...